apps/gateway/src/routes/app.ts

```diff
@@ -21,24 +22,16 @@ app.get('/health', (c) => {
   return c.json({ status: 'ok' })
 })
 
+configureOpenAPI(app)
+
+// Middlewares
 app.use(rateLimitMiddleware())
 app.use(authMiddleware())
 
-// Routers
-// v1
-app.route(
-  '/api/v1/projects/:projectId/versions/:versionUuid/documents',
-  documentsRouterV1,
-)
-app.route('/api/v1/conversations', conversationsRouterV1)
-
-// v2
-app.route(
-  '/api/v2/projects/:projectId/versions/:versionUuid/documents',
-  documentsRouterV2,
-)
-app.route('/api/v2/conversations', conversationsRouterV2)
-app.route('/api/v2/otlp', otlpTracesRouter)
+app.route('/', v1Routes)
+app.route('/', documents)
+app.route('/', conversations)
+app.route('/', telemetry)
```

Comment:
hmm this is very painful for the reader no? now there's no easy way to know what routes we are exposing you'd have to check every router

Reply:
What I would have like is something similar to rails where routes.rb declares all the routes and routers are just the controllers with the handlers for each endpoint in that route

Reply:
You can see all the routes in one place in `routes.ts`. Here are the paths

<img width="805" alt="image" src="https://github.com/user-attachments/assets/b5b51f6f-827f-46e9-ad95-5ceb2bd9377b" />